src/main.py

```diff
@@ -0,0 +1,36 @@
+from follower import Follower
+from leader import Leader
+from tagrec import TagRecognition
+
+
+"""
+Script to determine leader and follower roles.
+
+Takes input from the camera to determine if the current vehicle 
+should lead or follow based on whether or not there exists an ARTag
+in front of the vehicle.
+
+If an ARTag exists, the vehicle will become a follower vehicle.
+If an ARTag does not exist, the vehicle will become the leader vehicle.
+"""
+def main():
+    tag = TagRecognition()
+    
+
+    """
```

Comment:
These are docstrings. I meant comments like

\# If an ARTag is detected, there is a vehicle in front of this vehicle so it should follow them.
\# If an ARTag is not detected, there is no vehicle in front of this vehicle so it should be a leader.

Again, those are just quick thoughts. Change the language however you like.

Reply:
ah. gotcha

Reply:
pep8 online check doesn't like long comments, (above 79 characters) it is a bit difficult to get it to fully describe functionality within that limit for a single line.